### DISCUSSION TITLE
Cyberpunk 2077 not starting on Windows OpenZFS (Linux is fine)

### DISCUSSION THREAD

**Discussion by Zylvo** [openzfs/zfs#16911] [General]
Hi,
I didn't opened this as an issue, but I'm wondering where I can obtain logs or some debug info why it's not starting.
On Linux everything is ok, but on windows when trying to start cp2077 it displays error that there are corrupted or missing archives. No error log is being generated and after next start of the game, steam is doing verifications that says that all files are fine. I guess there's a problem with game accessing files way that ZFS is making problems with, but I wonder if there's any solution to it.

NTFS is fine. I've transferred my steam library from NTFS to ZFS and this is first game that I've issue with :(

@edit
I see that Skyrim is not downloading Creations and complaining that it's lacking permissions to write.

**Comment by IvanVolosyuk** [comment]
How do you access your ZFS files from windows? I have Steam/Cyberpunk 2077 shared as samba network drive with ZFS under the hood. It works fine with my windows 11 VM. Did you map the network drive to a drive letter? You may want to check file permissions and samba configuration.

**Reply by Zylvo** [reply]
Discussion went by mistake to openzfs/zfs instead of ofpenzfsonwindows/openzfs. Nevertheless, answering question: I access it using openzfs implementation for windows so I use it directly not as a network drive. There seems to be more issues with that, probably read/write handling, as cp2077 is not launching, Skyrim is complaining about lack of permission to write when trying to download mods from creation club, dawn of war soulstorm when selecting map to play is displaying them as two entries instead of one if they are stores as individual files (when playing vanilla, so they're stored inside a one compressed game file, no issues). On Linux those issues are not occurring so definitely something with windows handling of ZFS drive with openzfs driver.

**Reply by IvanVolosyuk** [reply]
Sounds like case sensitive file paths, you may want to check case sensitivity property and probably ask this question on openzfsonwindows/openzfs.

**Reply by Zylvo** [reply]
no, it was created with "-O casesensitivity=insensitive -O compression=zstd -O atime=off -o ashift=12 -o autotrim=on -O dedup=off -O xattr=sa -O acltype=nfsv4 -O sync=standard -O normalization=formD -O utf8only=on -O aclinherit=passthrough" and I've already opened an issue to openzfs windows GitHub as this looks bad for apps/games on windows driver or someone with more knowledge will correct me with some option that need to be adjusted or feature disabled/enabled :D

**Comment by n0xena** [comment]
btw - crosspost: https://github.com/openzfsonwindows/openzfs/issues/426
